src/main/resources/cms.conf

```diff
@@ -55,6 +55,10 @@ service.version="@@RELEASE@@"
 # This sets the maximum number of times a token can be refreshed
 auth.token.maxRefreshCount=24
 
+# Cache time for auth tokens in seconds.  This value limits the calls to AWS KMS service.
+# This value should be a small fraction of cms.iam.token.ttl.override (default is 1h)
+cms.iam.token.cache.ttl=60
```

Comment:
Would it make any sense to have the unit in the key, like in_seconds or something?